hutch_python/epics_arch.py

```diff
@@ -219,6 +280,7 @@ def pull_cds_data(exp, run):
         elif re.match('pcdssetup-temp.*-name', k):
             pv = cds_dict.get(re.sub('name', 'pvbase', k), '')
             displayList.append(QStruct(v, pv, "temperature"))
+    # logger.debug("displayList", displayList)
```

Comment:
nitpick: hanging comment, it's usually ok to leave debugs uncommented (unless this is huge?)

Reply:
If it's huge and you feel like you may need to bring it back frequently it's ok to keep it